spec/controllers/password_resets_controller_spec.rb

```diff
@@ -29,7 +29,7 @@
         alice = Fabricate(:user, password: "old_password")
         alice.update_column(:token, "12345")
         post :create, token: "12345", password: "new_password"
-        expect(alice.reload.authenticate("new_password")).to be_true 
+        expect(alice.reload.authenticate("new_password")).to be true
```

Comment:
This is the source of one of your failing tests. What you actually want here is the new syntax `be_truthy`